src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -2,34 +2,123 @@
 
 import frc.robot.Const;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj.I2C;
+
+import com.revrobotics.CANPIDController;
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
 
 import com.revrobotics.ColorSensorV3;
+import com.revrobotics.ControlType;
 
 /**
  * The Elevator subsystem controls the elevator mechanism
  *
- * This subsystem consists of the following components:
- * - The elevator motor (1x Spark Max using internal NEO encoder.)
- * - The adjustment wheel (1x Talon SRX/Victor SPX controller on either CAN or PWM)
- * - The color sensor for the color wheel
+ * This subsystem consists of the following components: - The elevator motor (1x
+ * Spark Max using internal NEO encoder.) - The adjustment wheel (1x Talon
+ * SRX/Victor SPX controller on either CAN or PWM) - The color sensor for the
+ * color wheel
  *
- * This subsystem should provide the following functions:
- * - Run a position loop on the elevator 
- * - Run a speed loop on the elevator
- * - Run a speed loop on the adjustment wheel
- * - Run a position loop for the color wheel (both matching color and rotations)
+ * This subsystem should provide the following functions: - Run a position loop
+ * on the elevator - Run a speed loop on the elevator - Run a speed loop on the
+ * adjustment wheel - Run a position loop for the color wheel (both matching
+ * color and rotations)
```

Comment:
What the heck did you do here? I'm guessing you did a format document.

Reply:
Yup, exactly. Didn't even catch this. Probably want to reject this.

Reply:
Haha yeah... but we may decide to merge the rest.